integration/travis/run_integration.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ev
+
+export PROJECT_DIR=`pwd`
+$PROJECT_DIR/../travis/start_scheduler.sh
+sleep 60
```

Comment:
@me-code-pretty Cook (running as an external process during this test) just takes a while to start up.  @dposada made a good suggestion of just polling to check the api's availability before starting the tests (this is what is done in the Simulator test).